x-pack/plugins/endpoint/common/types.ts

```diff
@@ -155,6 +161,20 @@ export interface EndpointEvent {
   agent: {
     type: string;
   };
+  endgame: {
```

Comment:
This interface is for the new elastic endpoint events which will not have any references to `endgame`. I think these belong in `LegacyEndpointEvent`.